projects/openapi-e2e-test/src/tests/errors/BookStoreFetchClientAdapter.ts

```diff
@@ -0,0 +1,11 @@
+import { FetchClientAdapter } from '@oats-ts/openapi-fetch-client-adapter'
```

Comment:
@erictheswift This is what I can offer in terms of unified error handling. This way it's still an internal implementation of the library, but you can get around it if you really want to.

The `Issue` array you get here is a representation of an error, here is the type definition:
https://github.com/oats-ts/oats-ts/blob/f5587ac33b55a3b8694bf81f6f334a1d2a5f75b6/projects/validators/src/typings.ts#L21